draft-ietf-httpbis-semantics-latest.xml

```diff
@@ -848,6 +851,151 @@ Content-Type: text/plain
 </artwork>
 </section>
 
+<section title="Origin" anchor="http.origin">
+<t>
+   The origin for a given URI is the triple of scheme, host, and port
+   after normalizing the scheme and host to lowercase and normalizing the
+   port to remove any leading zeros. If port is elided from the URI, the
+   default port for that scheme is used. For example, the URI
+</t>
+<artwork type="example">
+   https://Example.Com/happy.js
+</artwork>
+<t>
+   would have the origin
+</t>
+<artwork type="example">
+   { "https", "example.com", "443" }
+</artwork>
+<t>
+   which can also be described as the normalized URI prefix with port always
+   present:
+</t>
+<artwork type="example">
+   https://example.com:443
+</artwork>
+<t>
+   The notion of an origin for any given URI is central to the semantics of
+   HTTP. Each origin defines its own namespace and controls how identifiers
+   within that namespace are mapped to resources. In turn, how the origin
+   responds to valid requests, consistently over time, determines the
+   semantics that users will associate with a URI, and the usefulness of
+   those semantics is what ultimately transforms these mechanisms into a
+   "resource" for users to reference and access in the future.
```

Comment:
What do we mean by "users" in this context?  Do you refer to the user behind the user agent?  (In which case, this is fine and you don't need to make any changes.)